gpflow/inducing_variables/inducing_variables.py

```diff
@@ -57,7 +80,15 @@ def __init__(self, Z: TensorData, name: Optional[str] = None):
 
     @property
     def num_inducing(self) -> Optional[tf.Tensor]:
-        return tf.shape(self.Z)[0]
+        cs = ShapeChecker().check_shape  # mypy doesn't support decorating properties.
```

Comment:
Is this better than using a decorator and adding a type ignore?

Reply:
> Is this better than using a decorator and adding a type ignore?

Huh, `# type: ignore` hadn't even occurred to me. I'll do that. Thanks!